UI/UserLoginTest.py

```diff
@@ -0,0 +1,33 @@
+class UserLoginTest(TestCase):
+    def setUp(self):
+        self.client = User()
+
+    def test_administration_login(self):
```

Comment:
It is best practice to have a single assert statement per test.

Reply:
Thank you